modules/nf-core/untar_files/main.nf

```diff
@@ -0,0 +1,63 @@
+process UNTAR_FILES {
+    tag "$archive"
+    label 'process_single'
+
+    conda "conda-forge::sed=4.7 bioconda::grep=3.4 conda-forge::tar=1.34"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
+        'ubuntu:20.04' }"
+
+    input:
+    tuple val(meta), path(archive)
+
+    output:
+    tuple val(meta), path("${prefix}/**") , emit: files, optional: true
+    path "versions.yml"                   , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args  = task.ext.args ?: ''
+    def args2 = task.ext.args2 ?: ''
+    prefix    = task.ext.prefix ?: ( meta.id ? "${meta.id}" : archive.baseName.toString().replaceFirst(/\.tar$/, ""))
```

Comment:
I think we can assume an id will always be in the `meta`?

Reply:
If you say so, that was a call made by my predecessors here :-)

Reply:
I just think it's a bit weird to do it here as it's done nowhere else

Reply:
I agree :p especially since this is a universally used one

Reply:
Done- and tests corrected accordingly